manim/_config/plugins_flags.py

```diff
@@ -0,0 +1,42 @@
+"""
```

Comment:
I wouldn't put this file inside `_config`. As far as I can tell, this file contains functionality that can only be accessed via command line interface (and, I assume, that is the reason why it is within _config, no?) If so, I would put these functions inside _config/main_utils.py, or I would put this file inside manim/plugins.

Is there any other reason why it is here that I am missing? :thinking:

Reply:
Nothing you're missing. I'm just new to argparse logic :)